oshi-core/src/main/java/oshi/software/os/OSProcess.java

```diff
@@ -63,10 +63,7 @@
      * {@link #getArguments()} which already parses the results, and use this method
      * as a backup.
      * <p>
-     * On Linux and macOS systems, the string is null-character-delimited, to permit
-     * the end user to parse the executable and arguments if desired. This
-     * null-delimited behavior may change in future versions and should not be
-     * relied upon; use {@link #getArguments()} instead.
+     * On Linux and macOS systems, the string is space-character-delimited.
```

Comment:
All the OS's are now space-character delimited so you don't need to call out Linux and macOS separately here.  I'd just completely delete this section since the paragraph above indicates the necessary information for the user.